src/components/PizzaCanvas/PizzaCanvas.js

```diff
@@ -0,0 +1,126 @@
+import Component from '../../Component';
+import {PizzaImagery} from "../../services/PizzaImagery";
+import {CANVAS} from './settings';
+
+/**
+ * Class representing pizza constructed image.
+ * host is a <canvas> element
+ */
+export default class PizzaCanvas extends Component {
+  constructor(props) {
+    super(props);
+    this.ctx = this.host.getContext("2d");
+
+    this.host.width = CANVAS.px.width;
+    this.host.height = CANVAS.px.height;
+  }
+
+  /**
+   * Render view
+   * @returns {HTMLElement|string}
+   */
+  render() {
+    this.ctx.clearRect(0, 0, CANVAS.px.width, CANVAS.px.height);
+    // this.ctx.fillRect(10,10,150,150);
+    console.log(this.name + '.render() state', this.state);
+    this.drawPizzaBase();
+    this.drawIngredients();
+    return '';
+  }
+
+  /**
+   * Draws ingredients
+   */
+  drawIngredients() {
```

Comment:
It will be better if ingredients will scale according to pizza size, at the same way as pizza crust do.